### DISCUSSION TITLE
Bitrate Output Difference for Twitch and Youtube

### DISCUSSION THREAD

**Discussion by SirBassi** [datarhei/restreamer#748] [Q&A]
Hi together, first of All: What a awesome project! Makes fun to study all the possibillities in your detailed Guides, test own settings and enjoy the results. 

I am at a point where everything works smooth and fine but I am sruggling a bit to the Details:
**Scenario:** 
- I wanna send a fully mixed Scenary from OBS to Restreamer on my own Webserver
- From there I wanna "outroll" the Stream to Youtube and Twitch and own Website

**Problem:** 
- Twitch allows a Bitrate of 6000kbps
- Youtube allows a Bitrate of 10000kbps

So I wanna send from OBS with the higher Bitrate to Restreamer but how or is Restreamer managing then that Twitch allows just 6000kbps? I don't wanna risk that Twitch "caps" and give me a penalty for the high Bitrate and sets the Stream to 720p etc. 

Is Restreamer managing this Bitrate Differences "magically" in the Background or do I have spent some Attention to some Settings?

Thank u so much for your help.

**Comment by ioppermann** [comment]
If you need a different bitrate for a specific platform, then you need to re-encode the stream in that publication service in the "Source & encoding" section.

From OBS you send in the highest wanted bitrate. If Twitch allows only lower bitrates, then you need to re-encode the stream for that destination.
